packages/kbn-eslint-plugin-eslint/rules/no_deprecated_authz_config.js

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+const routeMethods = ['get', 'put', 'delete', 'post'];
+const ACCESS_TAG_PREFIX = 'access:';
+
+const isStringLiteral = (el) => el.type === 'Literal' && typeof el.value === 'string';
+const isLiteralAccessTag = (el) => isStringLiteral(el) && el.value.startsWith(ACCESS_TAG_PREFIX);
+const isLiteralNonAccessTag = (el) =>
+  isStringLiteral(el) && !el.value.startsWith(ACCESS_TAG_PREFIX);
+
+const isTemplateLiteralAccessTag = (el) =>
+  el.type === 'TemplateLiteral' && el.quasis[0].value.raw.startsWith(ACCESS_TAG_PREFIX);
+const isTemplateLiteralNonAccessTag = (el) =>
+  el.type === 'TemplateLiteral' && !el.quasis[0].value.raw.startsWith(ACCESS_TAG_PREFIX);
+
+const maybeReportDisabledSecurityConfig = (node, context, isVersionedRoute = false) => {
+  const callee = node.callee;
+  const isAddVersionCall =
+    callee.type === 'MemberExpression' && callee.property.name === 'addVersion';
+
+  const disabledSecurityConfig = `
```

Comment:
Do you think there's any utility in making the default enabled?  I'm imaging in the (admittedly unlikely) worst case where an auto-merge + already approved PR auto-fixes the issue.

Reply:
you mean enabled security? unfortunately, there are many routes with disabled security now and this is the default behavior